src/components/Header.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {useAppDispatch} from '../app/hooks';
+import useOpenFullScreen from '../hooks/useOpenFullScreen';
+import {ActionTypes} from '../store/constants';
+import {FullScreen, HeaderConsole, Logo, Logout, Sides, Title, UserInfo} from '../style/components/HeaderConsole';
+
+function Header() {
+  const dispatch = useAppDispatch();
+
+  const handleLogout = () => {
+    dispatch({
+      type: ActionTypes.LOGOUT,
+    });
+  };
+
+  const {fullScreen, openFullScreen} = useOpenFullScreen();
+
+  return (
+    <HeaderConsole>
+      <Sides>
+        <Logo src="/icons/logo.svg"></Logo>
+        <Title>API-консолька</Title>
+      </Sides>
+      <Sides>
+        <UserInfo>
+          some@email.com <span>:</span> sublogin
+        </UserInfo>
+        <Logout onClick={handleLogout}>
+          <span>Выйти</span>
+          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <g opacity="0.8">
+              <path
+                d="M9 21H5C4.46957 21 3.96086 20.7893 3.58579 20.4142C3.21071 20.0391 3 19.5304 3 19V5C3 4.46957 3.21071 3.96086 3.58579 3.58579C3.96086 3.21071 4.46957 3 5 3H9"
+                stroke="#0D0D0D"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+              <path d="M16 17L21 12L16 7" stroke="#0D0D0D" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" />
+              <path d="M21 12H9" stroke="#0D0D0D" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" />
+            </g>
+          </svg>
+        </Logout>
+        <FullScreen onClick={openFullScreen}>
+          {!fullScreen ? (
+            <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M6 1H3C2.46957 1 1.96086 1.21071 1.58579 1.58579C1.21071 1.96086 1 2.46957 1 3V6M19 6V3C19 2.46957 18.7893 1.96086 18.4142 1.58579C18.0391 1.21071 17.5304 1 17 1H14M14 19H17C17.5304 19 18.0391 18.7893 18.4142 18.4142C18.7893 18.0391 19 17.5304 19 17V14M1 14V17C1 17.5304 1.21071 18.0391 1.58579 18.4142C1.96086 18.7893 2.46957 19 3 19H6"
+                stroke="#0D0D0D"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+            </svg>
+          ) : (
```

Comment:
Please format ternary operator. Each condition should be from a new line. Here is the example. 
![image](https://user-images.githubusercontent.com/52970383/136940077-10fca8a3-6dec-45f1-8c81-42b2682e1287.png)